api-misc/src/main/java/org/zalando/nakadi/controller/SettingsController.java

```diff
@@ -111,7 +116,14 @@ public ResponseEntity<?> updateAdmins(@Valid @RequestBody final ResourceAuthoriz
         if (errors.hasErrors()) {
             throw new ValidationException(errors);
         }
-        adminService.updateAdmins(authz.toPermissionsList(ADMIN_RESOURCE));
+        final var newAdmins = authz.toPermissionsList(ADMIN_RESOURCE);
+        final var oldAdmins = adminService.updateAdmins(newAdmins);
```

Comment:
I didn't know this endpoint exists :)  So I was editing this in the DB directly, which is not audited...